### PR TITLE
[PR] Update HTML output for single posts

### PR DESCRIPTION
**Pull request by jeremyfelt**

### CODE REVIEW

**Comment by jeremyfelt** [comment]
Filters will work once https://github.com/washingtonstateuniversity/WSUWP-spine-parent-theme/pull/377 is merged.

File: articles/post.php [hunk 1]
```diff
@@ -4,71 +4,109 @@
...
-				?><figure class="article-thumbnail"><a href="<?php the_permalink(); ?>"><?php spine_the_thumbnail_image(); ?></a></figure><?php
-			} elseif ( spine_has_featured_image() ) {
-				?><figure class="article-thumbnail"><a href="<?php the_permalink(); ?>"><?php the_post_thumbnail( 'spine-thumbnail_size' ); ?></a></figure><?php
+			if ( 0 < count( $categories ) ) {
+				echo '<p class="meta-head meta-topic">Topics</p>';
+				echo '<ul class="meta-item-list">';
+
+				foreach ( $categories as $category ) {
+					$url = get_category_link( $category->term_id );
+					echo '<li class="meta-item"><a href="' . esc_url( $url ) . '">' . esc_html( $category->name ) . '</a>';
```
Comment (philcable) [inline articles/post.php]: Looks like the closing `</li>` is missing here.

File: articles/post.php [hunk 2]
```diff
@@ -4,71 +4,109 @@
...
-	<footer class="article-footer">
+<?php } else { // If not is_single() ?>
+	<header class="article-header">
+		<h2 class="article-title">
+			<a href="<?php the_permalink(); ?>" rel="bookmark"><?php the_title(); ?></a>
+		</h2>
+	</header>
+<?php } ?>
+
+<?php if ( ! is_singular() ) : ?>
```
Comment (philcable) [inline articles/post.php]: I'm probably missing something, but it looks like the code in this condition could be consolidated under the `else` just above?

**Review by philcable — CHANGES_REQUESTED** [review]
Looks good! I caught a missing closing tag that should be added before merging.

File: articles/post.php (hunk 1 above)
Comment (jeremyfelt) [reply articles/post.php]: Oops, nice catch!

File: articles/post.php (hunk 2 above)
Comment (jeremyfelt) [reply articles/post.php]: Ha, I thought the same thing but then didn't want to mess with anything. I'm going to remove it for clarity and then we'll see if anything pops up.

**Comment by jeremyfelt** [comment]
@philcable #reviewmerge part 2? :)

**Review by philcable — APPROVED** [review]
👍

**Pull request merged**